src/test/resources/json/cucumber-v3-with-screenshot.json

```diff
@@ -0,0 +1,134 @@
+[
```

Comment:
if you have more than one case, merge them all into sample.json

Reply:
Done.